docs/rms_oneliners.rst

```diff
@@ -0,0 +1,40 @@
+RMS "one-liner" functions
+=========================
+
+For lowerering the user threshold, some "one-liner" functions have been made for RMS. The purpose
+is both to make it simpler for users to export certain items, and in addition secure a better
+consistency. Hence the end user is not burdened to provide details, and only a script with quite
+a few lines will be needed.
+
+Exporting volumetrics from RMS
+------------------------------
+
+Volumetrics in RMS is always done by so-called volume jobs. The intention with the simplification
+is to use the RMS API behind the scene to retrieve all necessary data need for data io.
+
+Example:
+
+.. code-block:: python
+
+    from fmu.dataio.tools.rms import export_volumetrics
```

Comment:
Be more explicit on the module name? Would `fmu.dataio.exporters.rms` be better? In case we want other types of "tools" in the future?

Reply:
Should we trim it even further to `fmu.dataio.rms`?

`from fmu.dataio.rms import export_volumetrics    # since "export" is already a part of the function name`

Reply:
Learnings from early days of development is that if we start off creating something that assumes that we are in RMS, everything else becomes "exceptions" to that. And the amount of data export outside RMS is large and growing. Ideally, each one-liner should be available as both a Python function for direct usage (i.e. inside RMS) and as a corresponding ERT FORWARD_MODEL job for usage in that context.

If we trim it all the way down to `fmu.dataio.rms` we would still need to make `fmu.data.not-rms`. Not sure if that would work well?

Reply:
I landed on `fmu.dataio.export.rms`